core/src/test/java/com/novoda/merlin/receiver/ConnectivityCallbacksShould.java

```diff
@@ -0,0 +1,94 @@
+package com.novoda.merlin.receiver;
+
+import android.annotation.TargetApi;
+import android.net.ConnectivityManager;
+import android.net.Network;
+import android.net.NetworkInfo;
+import android.os.Build;
+
+import com.novoda.merlin.service.MerlinService;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Mock;
+
+import static org.fest.assertions.api.Assertions.assertThat;
+import static org.mockito.Mockito.*;
+import static org.mockito.MockitoAnnotations.initMocks;
+
+@TargetApi(Build.VERSION_CODES.LOLLIPOP)
+public class ConnectivityCallbacksShould {
+
+    private static final boolean CONNECTED = true;
+    private static final boolean DISCONNECTED = false;
+
+    private static final String ANY_REASON = "reason";
+    private static final String ANY_EXTRA_INFO = "extra info";
+
+    private static final int MAX_MS_TO_LIVE = 0;
+
+    @Mock
+    private ConnectivityManager connectivityManager;
+    @Mock
+    private MerlinService merlinService;
+    @Mock
+    private Network network;
+
+    private ConnectivityCallbacks networkCallbacks;
+
+    @Before
+    public void setUp() {
+        initMocks(this);
```

Comment:
This is something that needs to be addressed when I go through and boyscout the tests in the project. Done like this to match because we need to update too!